__snapshots__/chart-yaml.js

```diff
@@ -0,0 +1,5 @@
+exports['Chart.yaml updateContent updates version in Chart.yaml 1'] = `
+version: 1.1.0
+
```

Comment:
should this snapshot also contain `name:` like in your fixture.

Reply:
Updated updaters fixtures files